src/pybind/mgr/orchestrator.py

```diff
@@ -1221,15 +1221,15 @@ class StatelessServiceSpec(object):
     # start the services.
 
     def __init__(self, name, placement=None):
-        self.placement = PlacementSpec() if placement is None else placement  # type: PlacementSpec
+        self.placement = PlacementSpec(count=1) if placement is None else placement  # type: PlacementSpec
```

Comment:
My experience is, the orchestrator.py is the wrong place to invent _any_ defaults. Instead if we need defaults, we should do them in the cephadm/module.py module.

Reply:
OK, fair enough. However, line 1232's `else 1` never fires, and we have some defaulting going on. And then I tried to use -i in my vstart it would fail with a count being null. So just moved to it up here. Placement object actually gets recreated in the RGW spec after this anyway... so maybe more clean up is in order.